packages/client/src/app/direct-debit/page.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * @author David Zhu <dzhu292@aucklanduni.ac.nz>
+ */
+
+"use client";
+
+import Heading from "@/components/Heading/Heading";
+import PaymentInfoCard from "@/components/PaymentInfoCard/PaymentInfoCard";
+import DebitDetailsCard from "@/components/DebitDetailsCard/DebitDetailsCard";
+import Button from "@/components/Button/Button";
+
+export default function DirectDebitPage () {
+    const firstName:string = "John"
+    const lastName:string = "Smith"
+    const sessionId:string = "rn3498"
+    const accountNumber:string = "xx-xxxx-xxxx-xxx"
+
+    return (
+        <div className="h-[90vh] flex flex-col p-2"> {/* 90vh accounts for address bar on ios and etc */}
```

Comment:
I think this creates the unwanted side effect of causing the button to be quite far off the bottom of the screen when viewing in a screen like this. Do you have any thoughts on that?

<img width="438" alt="image" src="https://user-images.githubusercontent.com/74809724/236660848-62eebb4f-a34b-4ed8-98d6-4ab4c9b5bc07.png">

Reply:
Not too sure, we can either increase the vh a bit more or just change it back to how it was before (but that reintroduces the scrolling issues). One thing to note is that decreasing the div height of the screen only fixes the scrolling issue on some phones, but there will still be scrolling on smaller phones.

The heading currently takes up a lot of real estate so the size and padding of that might be something to reconsider.  

Also, the viewport takes the address bar into part of the screen height calculation so I think generally that bit of leeway in space may be necessary. If we push the elements to the very bottom of the page, then on a mobile browser it looks like this:
![IMG_1908](https://user-images.githubusercontent.com/119282717/236668924-f1427cf1-449b-4cc8-8c3e-d1a90adbe22c.png)
And on other browsers, it's just not visible.
![IMG_1909](https://user-images.githubusercontent.com/119282717/236668927-2dc431c5-c5cf-49d9-b364-398156cebdc6.png)

90vh for the div height makes this better, although still not perfect
![IMG_1910](https://user-images.githubusercontent.com/119282717/236669035-43d9d7e9-fc93-4be7-acf6-19342d966d77.png)
![IMG_1911](https://user-images.githubusercontent.com/119282717/236669040-6e529036-c730-405c-883e-c1eda2728cfe.png)

I think it could be interesting to use the sticky position tag and with some adjustments might be better.
https://imgur.com/a/1NAxpse

We could also use more responsive design techniques and make the size of the font and cards smaller according to screen size? Although I'm not sure how hard this will be to implement and test.

Reply:
Tagging @Harshal-D to keep him in the loop and for him to comment on this.